fe1-web/src/features/social/screens/SocialHome.tsx

```diff
@@ -52,46 +60,42 @@ const SocialHome = (props: IPropTypes) => {
       });
   };
 
-  const chirps = useMemo(makeChirpsList, []);
-  const chirpList = useSelector(chirps);
+  const chirpList = useSelector(makeChirpsList(laoId.valueOf()));
```

Comment:
is there a reason not to memo which eludes me ?

Reply:
I tried something that didn't work but I found a way to keep the useMemo, thanks for pointing this out!
Actually, I have to say I don't really understand its usage there, since `useSelector(makeChirpsList(laoId.valueOf()))` will already update the list when there is a change and we didn't use it before Nico refactored this. Maybe you could explain this to me, as what I found online lacks a little bit of clarity.

Reply:
Sure, let's talk about this offline. Github's a bit cumbersome :-)